.github/workflows/test.yaml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Build Docker image
         run: eval "$BUILD"
       - name: Mutation tests
-        timeout-minutes: 8
+        timeout-minutes: 15
```

Comment:
Did we really need to bump it this much? or was it a one-time test you were running?